frontends/p4/fromv1.0/programStructure.cpp

```diff
@@ -626,7 +626,8 @@ void ProgramStructure::createDeparser() {
 
 const IR::P4Table*
 ProgramStructure::convertTable(const IR::V1Table* table, cstring newName,
-                               IR::IndexedVector<IR::Declaration>* stateful) {
+                               IR::IndexedVector<IR::Declaration>* stateful,
+                               std::map<cstring, cstring> &mapNames) {
```

Comment:
I would give a more suggestive name to the mapNames argument, perhaps "actionRenameMap".